### ISSUE TITLE
Run on workerd

### ISSUE THREAD

**Issue by ignoramous** [serverless-dns/serverless-dns#90]
Ideally a statically linked workerd would be nice (https://github.com/cloudflare/workerd/issues/174), but it isn't available today.

Some folks ([1](https://github.com/Cyb3r-Jak3/docker-workerd), [2](https://github.com/frafra/workerd-docker/tree/fix-missing-libunwind)) have Dockerified it (https://github.com/cloudflare/workerd/issues/46), which fits nicely with deployments on Fly.

Some day, we switch to workerd in prod, but for now, perf seems to be lacking a bit: https://github.com/cloudflare/workerd/issues/72

**Comment by sr229** [comment]
Would using `deno compile` or `bun compile` work for your use case then? serverless-dns seems to be stateless so you can just package it as a static binary and run them at scale in your chosen provider.

**Comment by ignoramous** [comment]
True. I was interested in workerd mostly from a speed+efficiency point-of-view. I *imagine* bun is faster.

Deno works; bun, currently, doesn't.
- #72
